packages/botonic-plugin-knowledge-bases/src/hubtype-knowledge-api-service.ts

```diff
@@ -0,0 +1,37 @@
+import axios, { AxiosResponse } from 'axios'
+
+const DEFAULT_TIMEOUT = 10000
```

Comment:
are 10 seconds acceptable for our use case? which are the timings approximately of the calls to the knowledge api?

Reply:
the api is slow, it takes between 6 and 10 seconds, at user level it is slow when the knowledge base responds.